### PR TITLE
docs: synthetic-event branch

### PR DESCRIPTION
**Pull request by chloewlin**
7/6 Set up synthetic-event branch
7/6 This PR is ready for review

- [x] Overview
- [x] Event Pooling
- [x] Supported Events
- [x] Reference

### CODE REVIEW

File: content/docs/reference-events.md [hunk 1]
```diff
@@ -244,33 +244,33 @@ string pointerType
...
 ```
 
-A note on cross-browser support:
+關於跨瀏覽器支持的說明：
 
-Pointer events are not yet supported in every browser (at the time of writing this article, supported browsers include: Chrome, Firefox, Edge, and Internet Explorer). React deliberately does not polyfill support for other browsers because a standard-conform polyfill would significantly increase the bundle size of `react-dom`.
+目前為止，並非所有的瀏覽器都支持指標事件（在這篇文章書寫之時，支持該事件的瀏覽器有：Chrome, Firefox, Edge, and Internet Explorer）。 React 刻意不通過 polyfill 的方式支持其他瀏覽器，因為符合標準的 polyfill 會顯著地增加 `react-dom` 的 bundle 大小。
 
-If your application requires pointer events, we recommend adding a third party pointer event polyfill.
+如果你的應用程式需要指針事件，我們建議你加上第三方的指針事件 polyfill。
```
Comment (bugtender) [inline content/docs/reference-events.md]: I am not sure the "pointer event" is `指針事件` or `指標事件`(you use it in line 249).

Comment (neighborhood999) [reply content/docs/reference-events.md]: I prefer `指標`, but it should keep consistency

File: content/docs/reference-events.md [hunk 2]
```diff
@@ -244,33 +244,33 @@ string pointerType
...
 
 * * *
 
-### Selection Events {#selection-events}
+### 選擇事件 {#selection-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 3]
```diff
@@ -244,33 +244,33 @@ string pointerType
...
 onSelect
 ```
 
 * * *
 
-### Touch Events {#touch-events}
+### 觸摸事件 {#touch-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 4]
```diff
@@ -285,15 +285,15 @@ DOMTouchList touches
 
 * * *
 
-### UI Events {#ui-events}
+### UI 事件 {#ui-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 5]
```diff
@@ -302,15 +302,15 @@ DOMAbstractView view
 
 * * *
 
-### Wheel Events {#wheel-events}
+### 滾輪事件 {#wheel-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 6]
```diff
@@ -321,9 +321,9 @@ number deltaZ
 
 * * *
 
-### Media Events {#media-events}
+### 媒體事件 {#media-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 7]
```diff
@@ -334,25 +334,25 @@ onTimeUpdate onVolumeChange onWaiting
 
 * * *
 
-### Image Events {#image-events}
+### 圖片事件 {#image-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 8]
```diff
@@ -334,25 +334,25 @@ onTimeUpdate onVolumeChange onWaiting
...
 onLoad onError
 ```
 
 * * *
 
-### Animation Events {#animation-events}
+### 動畫事件 {#animation-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 9]
```diff
@@ -362,15 +362,15 @@ float elapsedTime
 
 * * *
 
-### Transition Events {#transition-events}
+### 過渡事件 {#transition-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 10]
```diff
@@ -380,9 +380,9 @@ float elapsedTime
 
 * * *
 
-### Other Events {#other-events}
+### 其他事件 {#other-events}
 
-Event names:
+事件名：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
事件名稱：
```

File: content/docs/reference-events.md [hunk 11]
```diff
@@ -31,19 +31,19 @@ number timeStamp
 string type
 ```
 
-> Note:
+> 注意：
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> 截至 v0.14 為止，從事件處理函式回傳 `false` 並不會停止事件冒泡（event propagation）。因此你可以選擇是情況手寫觸發 `e.stopPropagation()` 或 `e.preventDefault()`。
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
> 截至 v0.14 為止，從 event handler 回傳 `false` 並不會停止事件冒泡（event propagation）。因此你可以選擇是情況手寫觸發 `e.stopPropagation()` 或 `e.preventDefault()`。
```

File: content/docs/reference-events.md [hunk 12]
```diff
@@ -52,23 +52,23 @@ function onClick(event) {
...
+> 如果你想要用同步的方式讀取這些事件，你應該在該事件上呼叫 `event.persist()`。此方法將會把該合成事件從事件組合中移出，並允許使用者程式保留對該事件的引用。
 
-## Supported Events {#supported-events}
+## 支持的事件 {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React 將事件規格化，已讓它們在不同的瀏覽器中有ㄧ致的屬性。
 
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+以下的事件處理函式會在冒泡階段時被一個事件觸發。如果你想註冊捕獲階段的事件處理函式，請在事件名稱的後面加上 `Capture`。例如，假設你想在捕獲階段捕捉 click 事件的話，你需要用 `onClickCapture` 而不是 `onClick`：
```
Comment (neighborhood999) [inline content/docs/reference-events.md]:
```suggestion
以下的 event handler 會在冒泡階段時被一個事件觸發。如果你想註冊捕獲階段的 event handler，請在事件名稱的後面加上 `Capture`。例如，假設你想在捕獲階段捕捉 click 事件的話，你需要用 `onClickCapture` 而不是 `onClick`：
```

File: content/docs/reference-events.md [hunk 13]
```diff
@@ -91,31 +91,31 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
...
+屬性：
 
 ```javascript
 DOMDataTransfer clipboardData
 ```
 
 * * *
 
-### Composition Events {#composition-events}
+### 複合事件 {#composition-events}
```
[the remark this answers is not in the document]
Comment (chloewlin) [reply content/docs/reference-events.md]: Updated!

File: content/docs/reference-events.md [hunk 14]
```diff
@@ -91,31 +91,31 @@ The event handlers below are triggered by an event in the bubbling phase. To reg
 
 ## Reference {#reference}
```
[the remark this answers is not in the document]
Comment (chloewlin) [reply content/docs/reference-events.md]: Thanks. Updated!

File: content/docs/reference-events.md (hunk 1 above)
Comment (chloewlin) [reply content/docs/reference-events.md]: Updated!

**Comment by chloewlin** [comment]
Thank you, @bugtender and @neighborhood999, for your careful and thorough reviews. I've updated everything you suggested and will merge this in soon.

File: content/docs/reference-events.md [hunk 15]
```diff
@@ -1,18 +1,18 @@
 ---
 id: events
-title: SyntheticEvent
+title: 合成事件
```
[the remark this answers is not in the document]
Comment (chloewlin) [reply content/docs/reference-events.md]: Got it. Let's keep it then.

**Pull request merged**